modules/markup/markdown/markdown.go

```diff
@@ -278,7 +249,24 @@ func RenderString(ctx *markup.RenderContext, content string) (string, error) {
 
 // RenderRaw renders Markdown to HTML without handling special links.
 func RenderRaw(ctx *markup.RenderContext, input io.Reader, output io.Writer) error {
-	return render(ctx, input, output)
+	rd, wr := io.Pipe()
+	defer func() {
+		_ = rd.Close()
+		_ = wr.Close()
+	}()
+
+	go func() {
+		if err := render(ctx, input, wr); err != nil {
+			_ = wr.CloseWithError(err)
+			return
+		}
+		_ = wr.Close()
+	}()
+
+	buf := markup.SanitizeReader(rd, "")
+	_, err := io.Copy(output, buf)
```

Comment:
Since https://github.com/microcosm-cc/bluemonday/pull/110 has been merged we can avoid the buffer completely and just send the sanitization directly to the output.